conf/application.conf

```diff
@@ -150,8 +150,13 @@ feature-switch {
   }
 
   ifs_hip_migration_1502 {
-    enabled = true
-    released-in-production = false
+      enabled = true
+      released-in-production = false
+  }
+
+  ifs_hip_migration_1508 {
```

Comment:
Please update to `ifs_hip_migration_1508.enabled = true` as `release-in-production` is not applicable here, thanks.